src/cloudai/test_scenario_parser.py

```
@@ -117,10 +118,10 @@ def _parse_data(self, data: Dict[str, Any]) -> TestScenario:
         try:
             ts_model = TestScenarioModel.model_validate(data)
         except ValidationError as e:
-            logging.error(f"Failed to parse Test Scenario definition: {self.file_path}")
+            msg = f"Failed to parse Test Scenario definition: {self.file_path}"
             for err in e.errors(include_url=False):
-                err_msg = format_validation_error(err)
-                logging.error(err_msg)
+                msg += f"\n\t{format_validation_error(err)}"
+            logging.error(msg)
```

Ruff (0.15.10)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)